packages/cli/src/commands/base-command.ts

```diff
@@ -88,8 +88,11 @@ export abstract class BaseCommand extends Command {
 					instance: this.instanceSettings,
 				})
 			) {
-				// register module in the registry for the dependency injection
-				await import(`../modules/${moduleName}/${moduleName}.module`);
+				const resolvedModuleName = enterpriseModules.includes(moduleName)
```

Comment:
I'm wondering if adding more convention logic here is really necessary. Could we "just" have the module in `modules/external-secrets.ee/external-secrets.ee.module.ts` ?